eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -107,3 +107,13 @@ jobs:
             }
           failOnStderr: true
           pwsh: true
+
+  - job: Compliance
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+    steps:      
+      - template: /eng/common/pipelines/templates/steps/credscan.yml
+        parameters:
+          BaselineFilePath: $(Build.sourcesdirectory)/eng/dotnet.gdnbaselines
+          ServiceDirectory: ${{ parameters.ServiceDirectory }}
```

Comment:
ServiceDirectory is not a parameter to this template so this will always be missing. We probably need to make this a pass Scope through.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/28624/checks?check_run_id=7191038561

As indicate in the above check, it does not affect the result by passing empty here. We will check changed file in PR first. I will have another PR to remove the unnecessary parameter here.

Reply:
I already fixed this in https://github.com/Azure/azure-sdk-for-net/pull/29501